ci-operator/step-registry/ibmcloud/provision/kms-key/ibmcloud-provision-kms-key-commands.sh

```diff
@@ -24,6 +24,34 @@ function create_resource_group() {
     "${IBMCLOUD_CLI}" target -g ${rg} || return 1
 }
 
+function createKey() {
+    local instance_name=$1
+    local region=$2
+    local keyInfoFile=$3
+    local id key_name key_meterial tmpFile keyCRN keyID keyInfo
```

Comment:
I think we do not need `keyInfo`, we can directly write the info into `keyInfoFile`